### PR TITLE
Fixes #25209 - CV import/export docs

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+layout: plugins/katello/documentation
+title: Content View Import/Export
+version: nightly
+---
+
+#Content View Import/Export
```
Comment (ekohl) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: Missing space after `#` makes markdown sad

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+---
+layout: plugins/katello/documentation
+title: Content View Import/Export
+version: nightly
+---
+
+# Content View Import/Export
+
+Content view import/export is one of the new features in Katello 3.9. This feature is for users who have one Katello instance with Internet access, but have other instances without Internet access. Import/export allows users to get content view versions from the "connected" Katello to the "disconnected" Katello.
```
Comment (ehelms) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: I wouldn't limit this to purely non-internet access. As this is useful for anyone wanting to keep multiple Katello's in sync with one another. Perhaps word it rather as this is useful for those who have disconnected environments.

File: plugins/katello/nightly/user_guide/content_view_import_export/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+This new feature works differently than the old export/import feature. The old feature is still available but has been deprecated.
+
+In earlier versions of Katello, you could only export yum repositories. You could export an entire content view version, but this simply exported each yum repository in the version without any additional metadata.
+
+We wanted to give system administrators the ability to have fine-grained control over their content view versions, and allow them to have the same content view on two or more Katello instances. We created additional APIs to allow for this, and we added commands to the hammer CLI tool.
+
+We will first discuss the changes we made, and then discuss two common use cases.
```
Comment (ehelms) [inline plugins/katello/nightly/user_guide/content_view_import_export/index.md]: Given this is docs, I'd drop the `we` and aim for more of a third person voice.

**Review by ehelms — APPROVED** [review]

**Comment by beav** [comment]
@ehelms @ekohl thanks! merging

**Pull request merged**